plugins/discovery-azure-classic/build.gradle

```diff
@@ -88,9 +90,10 @@ task createKey(type: LoggedExec) {
     '-keypass', 'keypass',
     '-storepass', 'keypass'
 }
-
+//no unit tests
+test.enabled = false
```

Comment:
This seems like one step forward and one step back. I assume this is also just to appease our testing conventions stuff? If so I feel like this should not be a failure.

Reply:
yup.  However, it is a bit more rare to have no unit tests (at least until i start converting the qa projects).  For this one we will probably want to toggle the enable/disable the test task if the directory exists or not. I can look into that for a follow up PR.

Reply:
Yeah, we are going to have this all over the place for the qa projects that contain only rest tests.